src/fragments/lib/push-notifications/js/getting-started.mdx

```diff
@@ -91,13 +91,13 @@ Setup instructions are provided for Android and iOS, and configuration for both
     apply plugin: "com.google.gms.google-services"
     ``` 
 
-8. Open *android/gradle/wrapper/gradle-wrapper.properties* update the Gradle `distributionUrl`:
+8. Open *android/app/src/main/AndroidManifest.xml* file and add the following configuration into `application` element.
 
-    ```properties
-    distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-all.zip
-    ```
+    <Callout>
 
-9. Open *android/app/src/main/AndroidManifest.xml* file and add the following configuration into `application` element.
+    When using Android 12+, make sure to add the property `android:exported="false"` to the service tags below.
```

Comment:
I think we might be able to just add that property to our snippet. Android 12 began enforcing that this property be explicitly specified but there should be no harm (and is perhaps better practice) to include it regardless.

Reply:
Changed this to suggested